src/olympia/devhub/tasks.py

```diff
@@ -88,7 +89,9 @@ def create_version_for_upload(addon, upload):
 
         log.info('Creating version for {upload_uuid} that passed '
                  'validation'.format(upload_uuid=upload.uuid))
-        version = Version.from_upload(upload, addon, [amo.PLATFORM_ALL.id])
+        beta = is_beta(upload.version)
```

Comment:
I updated this to treat no version as not beta. I don't think we need to fail on this but I don't think it should happen either.

```py
beta = bool(upload.version) and is_beta(upload.version)